### PR TITLE
Add Docker support

### PR DESCRIPTION
**Pull request by pataquets**
Add Dockerfile to enable image building. Useful for development and tests under Docker workflow.
Using the official Python image, `:2` tag. More info at https://hub.docker.com/_/python/

Just adding files, setting working dir and running install.

Build:

```
$ docker build -t elasticstat .
```

Run:

```
$ docker run --rm -it elasticstat [options...]
```

FYI, there's a still quicker to test, already built image on my Docker Hub. Test it by running:

```
$ docker run --rm -it pataquets/elasticstat-src [options...]
```

Should stop by `CTRL+C`'ing it.
If no options given, assume `--help`to avoid any error message.

Optional improvement to come (maybe in another issue):
- Create an 'official', based on your repo, automated build at Docker Hub for the image: https://docs.docker.com/docker-hub/builds/ . Just requires a free Docker Hub account and a following a quick 'Create automated build' process. I'll be happy to help on it, if needed. Useful as an easy direct-from-source (branches and tags) distribution method and for having a 'daily' build from source on each commit.

### CODE REVIEW

**Comment by paulrossmeier** [comment]
Hello pataquets,

I wanted to thank you for your contribution - we are reviewing and will update accordingly.

Paul Rossmeier

**Comment by paulrossmeier** [comment]
I have tested this and works fine - I need to get a reviewer to approve the merge

Paul Rossmeier

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM python:2
```
Comment (naemono) [inline Dockerfile]: Mind using `python:2-slim` here instead, as the resulting docker image size difference is pretty huge

```
$ docker images | grep elasticstat
objectrocket/elasticstat                                python2-slim                               5a2e1166a522        5 seconds ago       156MB
objectrocket/elasticstat                                python2                                    9170d29364f8        2 minutes ago       899MB
```

Comment (pataquets) [reply Dockerfile]: Done

**Pull request merged**